openedx/core/djangoapps/schedules/signals.py

```diff
@@ -53,14 +56,22 @@ def create_schedule(sender, **kwargs):
 
     upgrade_deadline = _calculate_upgrade_deadline(enrollment.course_id, content_availability_date)
 
-    Schedule.objects.create(
+    schedule = Schedule.objects.create(
         enrollment=enrollment,
         start=content_availability_date,
         upgrade_deadline=upgrade_deadline
     )
 
-    log.debug('Schedules: created a new schedule starting at %s with an upgrade deadline of %s',
-              content_availability_date, upgrade_deadline)
+    try:
+        get_week_highlights(enrollment.course_id, 1)
+        experience_type = ScheduleExperience.COURSE_UPDATES
+    except CourseUpdateDoesNotExist:
+        experience_type = ScheduleExperience.DEFAULT
+
+    ScheduleExperience(schedule=schedule, experience_type=experience_type).save()
+
+    log.debug('Schedules: created a new schedule starting at %s with an upgrade deadline of %s and experience type: %s',
+              content_availability_date, upgrade_deadline, ScheduleExperience.EXPERIENCES[experience_type][1])
```

Comment:
Nit: If we make `ScheduleExperience.EXPERIENCES` a `namedtuple`, we wouldn't need to retrieve values using positional indices.

Reply:
I replaced this with the Choices utility from model_utils